docs/tests/core-concepts/client/plans/output_marshal.cue.fragment

```diff
@@ -0,0 +1,18 @@
+#SomeAction: {
```

Comment:
That was a good idea to simplify example!

Reply:
Following @shykes flow here :D